lib/app.js

```diff
@@ -283,8 +284,12 @@ function App () {
   app.post('/public/:collectionId/:displayId/:version/icon', requireUser, uploadToMemory.single('collectionIcon'), actions.updateCollectionIcon)
 
   app.get('/public/:collectionId/:displayId/:version/removeCollection', requireAdmin, actions.removeCollection)
-  app.get('/user/:userId/:collectionId/:displayId/:version/removeCollection', requireUser, actions.removeCollection)
+  app.get('/public/:collectionId/:displayId/:version/addToCollection', requireAdmin, views.addToCollection)
+  app.post('/public/:collectionId/:displayId/:version/addToCollection', requireAdmin, views.addToCollection)
 
+  app.get('/user/:userId/:collectionId/:displayId/:version/removeCollection', requireUser, actions.removeCollection)
+  app.get('/user/:userId/:collectionId/:displayId/:version/addToCollection', requireUser, views.addToCollection)
+  app.post('/user/:userId/:collectionId/:displayId/:version/addToCollection', requireUser, views.addToCollection)
```

Comment:
Do they need share endpoints?

Reply:
I was not sure if it made sense to allow share endpoints to have this capability or not.  I thought share endpoints were basically read-only, and add owner was used to give edit capability.